docs/disabled_features.md

```diff
@@ -5,9 +5,11 @@ sort_rank: 10
 
 # Disabled Features
 
-Here is a list of features that are disabled by default since they are breaking changes.
-They can be enabled using the `--enable-feature` flag and will be enabled by default
-in future when it's adopted widely.
+Here is a list of features that are disabled by default since they are breaking changes or are considered experimental.
+Their behaviour can change in the future releases which will be communicated via [release changelog](https://github.com/prometheus/prometheus/blob/master/CHANGELOG.md).
```

Comment:
via the

Reply:
in future

Its either in future releases, or in the future - both sounds weird.

Reply:
Done